src/EnergyPlus/PlantUtilities.cc

```diff
@@ -149,7 +149,13 @@ void SetComponentFlowRate(EnergyPlusData &state,
     Real64 const MdotOldRequest = state.dataLoopNodes->Node(InletNode).MassFlowRateRequest;
     auto &loop_side = state.dataPlnt->PlantLoop(plantLoc.loopNum).LoopSide(plantLoc.loopSideNum);
     auto const &comp = loop_side.Branch(plantLoc.branchNum).Comp(plantLoc.compNum);
-
+    if (DataPlant::CompData::getPlantComponent(state, plantLoc).FlowCtrl == DataBranchAirLoopPlant::ControlType::SeriesActive) {
+        Real64 seriesFlowVal = state.dataLoopNodes->Node(DataPlant::CompData::getPlantComponent(state, plantLoc).NodeNumIn).MassFlowRate;
```

Comment:
@EnergyArchmage What about downstream components on a series active branch. For example, 2 chillers in series, the first chiller is off, but the second chiller is on. How does the first chiller find out there should be flow on the branch?

Reply:
That situation is the main point the change is trying to address.   The beginning iteration there isn't any flow yet, so yes, the first chiller is off and doesn't see any flow on the node.   But the next iteration, the second chiller will come on and make a flow request, and if flow gets going then the first chiller will see flow on the node.   The idea is for the first chiller to _not_ keep requesting zero flow even when there is flow.  That zero request seems to set up a fight between a zero flow request and some flow and isn't helpful when we know there is already flow.   Series active branches are supposed to look at the last node on the branch but it seems that design is not completely implemented and mismatched flow requests on upstream nodes can cause problems, like more iterations before things settle down.

Reply:
Why not use `if (comp.FlowCtrl == DataBranchAirLoopPlant::ControlType::SeriesActive)` on line 152 and comp.NodeNumIn on line 153

    CompData &CompData::getPlantComponent(EnergyPlusData &state, PlantLocation const &plantLoc)
    {
        return state.dataPlnt->PlantLoop(plantLoc.loopNum).LoopSide(plantLoc.loopSideNum).Branch(plantLoc.branchNum).Comp(plantLoc.compNum);
    }

Reply:
It wasn't clear to me if that "comp" is a real pointer